dataRetriever/resolvers/headerResolver_test.go

```diff
@@ -315,6 +361,11 @@ func TestHeaderResolver_ProcessReceivedMessageRequestNonceTypeNotFoundInHdrNonce
 		&mock.CacherStub{},
 		headersNonces,
 		&mock.StorerStub{},
+		&mock.StorerStub{
+			GetCalled: func(key []byte) (i []byte, e error) {
+				return nil, nil
```

Comment:
I don't see any any use in returning nil, nil, maybe change to a NotFound error?

Reply:
This is actually a stub usage. 
It will be changed to return something else.

Reply:
Changed all occurrences.